doc-content/enterprise-only/admin-and-config/manage-log-file-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='manage-log-file-proc']
+= Managing log files
+
+{PRODUCT} manages the required maintenance, runtime data that is removed, including:
+
+* *Process instance data*, which is removed upon process instance completion.
+* *Work item data*, which is removed upon work item completion.
+* *Task instance data*, which is removed upon completion of a process to which the given task belongs.
+
+Runtime data, which is not cleaned automatically includes session information data that is based on the selected runtime strategy.
+
+* *Singleton strategy* ensures that runtime data of session information is not automatically removed.
+* *Per request strategy* allows automatic removal when a request is terminated.
+* *Per process instances* are automatically removed when a process instance is mapped to a session that is completed or aborted.
+
+In order to keep track of process instances, {PRODUCT} provides audit data tables. There are two ways to manage and maintain the audit data tables, including cleaning up the jobs <<setup-auto-cleanup-proc, automatically>> and <<manual-cleanup-con, manually>>.
```

Comment:
Add "the".
In order to keep **the** track of process instances,

Reply:
Done